### PR TITLE
WPF-62372-Updated-UG-documents

### PR DESCRIPTION
**Pull request by BhaskarSuresh22**
Updated the UG documents with Windows11 theme and .NET 6 contents and removed old contents.

### CODE REVIEW

**Review by jeganraj-m — APPROVED** [review]
changes are fine

File: wpf/Themes/ScrollBar-Mode.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ The [ScrollBarMode](https://help.syncfusion.com/cr/wpf/Syncfusion.SfSkinManager.
 * `Default` - The ScrollBar will be displayed with the default look of a thumb along with up and down arrow to scroll.
 * `Compact` - The Windows 10 like compact scrollbar will have the look of a thin layer of thumb until the user hovers over the scrollbar area. 
 
-N> The `Compact` scrollbar mode is enabled by default in [WPF Fluent Theme](https://help.syncfusion.com/wpf/themes/fluent-theme). 
+N> The `Compact` scrollbar mode is enabled by default in [WPF Fluent Theme](https://help.syncfusion.com/wpf/themes/fluent-theme) and WPF Windows11 Theme.
```
Comment (Chrispineagunja) [inline wpf/Themes/ScrollBar-Mode.md:18]: N> The `Compact` scrollbar mode is enabled by default in the [WPF Fluent Theme](https://help.syncfusion.com/wpf/themes/fluent-theme) and WPF Windows11 Theme.

File: wpf/dotnet-core-compatibility.md [hunk 2]
```diff
@@ -21,32 +21,44 @@ Below table represents the supported Syncfusion Essential Studio version for .NE
...
+From 20.4 (2022 Vol4) <br/></td>
+<td>No<br/></td>
+<td>No<br/></td>
+<td>Yes<br/></td>
 </tr>
 
 </table>
 
-N> For.NET 6.0 and higher versions, dedicated assemblies are not included. Instead, .NET 5.0 assemblies can be used.
+N> For.NET 7.0 and higher versions, dedicated assemblies are not included. Instead, .NET 6.0 assemblies can be used.
```
Comment (Chrispineagunja) [inline wpf/dotnet-core-compatibility.md:61]: N> Dedicated assemblies are not included for .NET 7.0 and higher versions. Instead, .NET 6.0 assemblies can be used.

File: wpf/dotnet-core-compatibility.md [hunk 3]
```diff
@@ -66,13 +78,8 @@ Refer [NuGet package](https://help.syncfusion.com/wpf/visual-studio-integration/
 You can also add the Syncfusion WPF (.NET Core) controls assemblies from `precompiledassemblies` folder in the following location.
```
Comment (Chrispineagunja) [inline wpf/dotnet-core-compatibility.md:78]: You can also add the Syncfusion WPF (.NET Core) controls assemblies from the `precompiledassemblies` folder in the following location.

Comment (BhaskarSuresh22) [reply wpf/dotnet-core-compatibility.md]: Done the changes

File: wpf/dotnet-core-compatibility.md (hunk 2 above)
Comment (BhaskarSuresh22) [reply wpf/dotnet-core-compatibility.md]: Done the changes.

File: wpf/Themes/ScrollBar-Mode.md (hunk 1 above)
Comment (BhaskarSuresh22) [reply wpf/Themes/ScrollBar-Mode.md]: Done the changes.

**Pull request merged**